### ISSUE TITLE
npm install on Windows shared folder

### ISSUE THREAD

**Issue by rkcreation** [artificialio/sane#58]
I try to use sane on a Windows shared folder (mounted with cifs), but I need to *npm install --no-bin-links* for a working install.

Do you have any idea for doing that ?

Thanks in advance

**Comment by Globegitter** [comment]
I have not tried that yet. Is it throwing any errors? But can hopefully test soon.

**Comment by rkcreation** [comment]
Example of error (there is many like that) : 
```
Installing Sails packages for tooling via npm....npm ERR! Linux 3.2.0-4-amd64
npm ERR! argv "node" "/usr/local/bin/npm" "i" "sails-generate-ember-blueprints" "lodash" "morgan" "pluralize" "sails-disk" "--save"
npm ERR! node v0.10.35
npm ERR! npm  v2.1.16
npm ERR! path ../rimraf/bin.js
npm ERR! code UNKNOWN
npm ERR! errno -1
```

I already have the problem for another node project, and add *--no-bin-links* to *npm install* works for me, I've found the solution here : http://askubuntu.com/questions/269727/npm-errors-when-installing-packages-on-windows-share

Is there any solution, or possible workarounds for specify some arguments to node install with the sane cli ? I'm ready for testing some code, or help in some way for this issue.

**Comment by Globegitter** [comment]
Oohh, you mean that you have something like `sane new project --no-bin-links`?

**Comment by rkcreation** [comment]
Yes, somthing like that, or maybe with the .sane-cli file, which let us specify various arguments to add to `npm install`, or maybe `sane new project --install="--no-bin-links"`

**Comment by Globegitter** [comment]
Oh yeah, what you can do in the meantime btw, is try `sane new project --skip-npm` and then you can install everything manually.
Plus once all packages are installed you would also need to run `sails generate ember-blueprints`
